server/middleware/validate.js

```diff
@@ -0,0 +1,41 @@
+class Midware{
+
+	isValid (req, res, next) {
+		let {location, comment} = req.body;
```

Comment:
please change this to a `const` since location and comment are not reassigned anywhere else.